plugins/pv_lxc.c

```diff
@@ -468,99 +442,14 @@ static void pv_setup_default_log(struct pv_platform *p, struct lxc_container *c,
 	}
 }
 
-/*
- * Use only after loading config.
- * Use threshold as 0 to always truncate.
- */
-static void pv_truncate_lxc_log(struct lxc_container *c,
-				const char *platform_name, off_t threshold,
-				const char *key)
-{
-	char logfile_name[PATH_MAX];
-
```

Comment:
this reminds me: do we have a truncation story in the new singlefile and filetree formats?

Reply:
we have some truncation configs in pantavisor that we would prefer to keep try honoring...